webapp/cube/views.py

```diff
@@ -193,12 +193,16 @@ def cube_microcerts():
 
             courses.append(course)
 
+    edx_register_url = f"{edx_url}{flask.request.base_url}"
+    if flask.request.args.get("test_backend"):
```

Comment:
This will also work with `flask.request.args.get("test_backend") == "false"`? Anyway, if only devs use it should be fine, as long as people know that they need to drop it entirely

Reply:
Valid question. It would not work correctly. If users do `test_backend=false`, the env would be still set to testing. 

It was just a quick thing. I am not very happy I had to check for the `test_backend` like this inside the view function either, but I did not know a better way around it.

I will take a look to see if I can move it inside the decorator somehow.